pallets/gear/src/schedule.rs

```diff
@@ -764,7 +764,7 @@ impl Default for Limits {
 impl<T: Config> Default for InstructionWeights<T> {
     fn default() -> Self {
         Self {
-            version: 10,
+            version: 11,
```

Comment:
@breathx, don't we rise version during releases?

Reply:
nothings criminal here
could be bumped here as well for test purposes. each time re-checked on releases